gapic/src/main/com/google/gapid/views/TracerDialog.java

```diff
@@ -262,6 +262,7 @@ protected void buttonPressed(int buttonId) {
       private static final int DEFAULT_START_FRAME = 100;
       private static final String PERFETTO_LABEL = "Profile Config: ";
       private static final String NO_GPU_PROFILING_CAPABILITY = "Warning: Selected device has no GPU profiling capability.";
+      private static final String NO_FRAME_LIFECYCLE_CAPABILITY = "Warning: Selected device has no Frame Lifecycle capability.";
```

Comment:
why show this warning? For GPU profiling we show the warning because we disallow usage of the tool if no GPU profiling capability is detected.

Reply:
The frame lifecycle events are available only from R onwards. This warning is for devices that have Q and lower versions of android.

Reply:
I agree, we don't need to show a warning in this case. Simply not showing the checkbox is enough.

Reply:
Removed it.

Reply:
still here.